### DISCUSSION TITLE
How long is it supposed to take for initial discovery?

### DISCUSSION THREAD

**Discussion by BQuinn-Assured** [aws-solutions/workload-discovery-on-aws#513] [General]
Successful deployment several hours ago but still no luck. Everything is up running and green. but I still see this.  
![image](https://github.com/aws-solutions/workload-discovery-on-aws/assets/155664915/b8d45c0c-27b1-409d-97bc-b6f421ba7c6b)

What other things is initial discovery dependent on? 

Did I do something wrong?

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
